### PR TITLE
python 2.7.13

### PR DESCRIPTION
**Pull request by zmwangx**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

I'm updating setuptools, pip and wheel to the latest versions. Also pulling out the embedded tk patch into formula-patches (https://github.com/Homebrew/formula-patches/pull/97), because it makes the formula cleaner. Let me know if you want it back.

### CODE REVIEW

File: Formula/python.rb [hunk 1]
```diff
@@ -43,38 +42,26 @@ class Python < Formula
...
-    url "https://files.pythonhosted.org/packages/e7/a8/7556133689add8d1a54c0b14aeff0acb03c64707ce100ecd53934da1aa13/pip-8.1.2.tar.gz"
-    sha256 "4d24b03ffa67638a3fa931c09fd9e0273ffa904e95ebebe7d4b1a54c93d7b732"
+    url "https://files.pythonhosted.org/packages/11/b6/abcb525026a4be042b486df43905d6893fb04f05aac21c32c638e939e447/pip-9.0.1.tar.gz"
+    sha256 "09f243e1a7b461f654c26a725fa373211bb7ff17a9300058b205c61658ca940d"
   end
 
   resource "wheel" do
-    url "https://files.pythonhosted.org/packages/c9/1d/bd19e691fd4cfe908c76c429fe6e4436c9e83583c4414b54f6c85471954a/wheel-0.29.0.tar.gz"
-    sha256 "1ebb8ad7e26b448e9caa4773d2357849bf80ff9e313964bcaf79cbf0201a1648"
+    url "https://files.pythonhosted.org/packages/a7/37/947b4329c4a3c72093b6c8e9b4be8c7f10c32dbb78848d3a234ce01c059d/wheel-0.30.0a0.tar.gz"
```
Comment (zmwangx) [inline Formula/python.rb]: Right. For some reason virtualenv picks up wheel 0.30.0a0, so poet generated this, and I didn't check...

Comment (zmwangx) [inline Formula/python.rb]: > For some reason virtualenv picks up wheel 0.30.0a0

Apparently this has been fixed in virtulenv 22.0.5: https://github.com/pypa/virtualenv/blob/15.1.0/docs/changes.rst#1510-2016-11-15.

**Comment by zmwangx** [comment]
Updated patch's commit hash. @tdsmith Might want to kill the CI builds #13367 and #13371.

**Review by tdsmith — APPROVED** [review]

**Comment by zmwangx** [comment]
Test of `yelp-tools` is failing, but that's because the XML schema http://projectmallard.org/1.0/mallard-1.0.rng can't be loaded (projectmallard.org is down at the moment); it shouldn't block the Python update.

**Comment by tdsmith** [comment]
Thanks Zhiming!

**Pull request closed without merging**